unit-tests/src/main/scala/tests/Suite.scala

```diff
@@ -16,12 +16,16 @@ abstract class Suite {
   def assertNot(cond: Boolean): Unit =
     if (cond) throw AssertionFailed else ()
 
-  def assertThrows[T: ClassTag](f: => Unit): Unit = {
+  def assertThrows[T: ClassTag](f: => Unit): Unit =
+    assertThrows2[T](f, _ => true)
+
+  def assertThrows2[T: ClassTag](f: => Unit, fe: T => Boolean): Unit = {
```

Comment:
`assertThrowsAnd` has a different method name, how is it classhing with `assertThrows`?